A patch that fakes the android version so that NNAPI can be loaded and used
by tensorflow on chrome OS.

This patch can be removed when
https://github.com/tensorflow/tensorflow/commit/2fd3f75f05b99cd2d3c36ac57384633e7707d307
is released.

diff --git a/tensorflow/lite/nnapi/nnapi_implementation.cc b/tensorflow/lite/nnapi/nnapi_implementation.cc
index 862c4ba2..e275f266 100644
--- a/tensorflow/lite/nnapi/nnapi_implementation.cc
+++ b/tensorflow/lite/nnapi/nnapi_implementation.cc
@@ -92,7 +92,7 @@ int ASharedMemory_create(const char* name, size_t size) {

 const NnApi LoadNnApi() {
   NnApi nnapi = {};
-  nnapi.android_sdk_version = 0;
+  nnapi.android_sdk_version = 30;

 #ifdef __ANDROID__
   nnapi.android_sdk_version = GetAndroidSdkVersion();
